### DISCUSSION TITLE
299

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#965] [Comments]
# 299

IntelliJ와 Spring Rest Docs를 사용하면 필수로 설치하는게 AsciiDoc Plugin입니다. (AsciiDoc Plugin) 마크다운 Preview처럼 AsciiDoc 파일을 미리 볼 수 있도록 지원하는 아주 좋은 플러그인입니다. 하지만 Spring Rest Docs에서는 활용하기가 쉽지 않습니다. 일반적으로 Spring Rest Docs는 아래처럼 {snippets}을 기준으로 Import 할 adoc 파일 위치를 지정합니다.하지만 이렇게 {snippets} Path를 제대로 인식하지 못합니다. 아시다시피 {snippets}는 Maven/Gradle이 html파일을 만들 때 교체될 변수이기 때문입니다. build 후에 굳이 html파일에서 보지 않고, adoc 파일에서 바로 결과물을..

https://jojoldu.tistory.com/299

**Comment by seoyongsu** [comment]
향로님 항상 기술들을 정리해주셔서 보고 많이 활용하면서 응용하고 있습니다.
너무 감사합니다.
